.github/workflows/rawls-build-tag-publish-and-run-tests.yaml

```diff
@@ -0,0 +1,135 @@
+name: rawls-build-tag-publish-and-run-tests
+
+on:
+  push:
+    branches:
+      - DDO-2380
```

Comment:
Presume this branch is going away after you merge this?

Reply:
Yes this will go away before merge. It's used for testing purpose.